migrations/update_filtered_positives.py

```diff
@@ -0,0 +1,101 @@
+from crawler.helpers import get_config
+from datetime import datetime
+from migrations.helpers.update_filtered_positives_helper import (
+    pending_plate_barcodes_from_dart,
+    positive_result_samples_from_mongo,
+    update_filtered_positive_fields,
+    update_mongo_filtered_positive_fields,
+    update_mlwh_filtered_positive_fields,
+    update_dart_filtered_positive_fields,
+)
+from migrations.helpers.shared_helper import print_exception
+from crawler.filtered_positive_identifier import FilteredPositiveIdentifier
+
+
+def run(settings_module: str = "") -> None:
+    """Updates filtered positive values for all positive samples in pending plates
+
+    Arguments:
+        config {ModuleType} -- application config specifying database details
+    """
+    print("-" * 80)
+    print("STARTING FILTERED POSITIVES UPDATE")
+    print(f"Time start: {datetime.now()}")
+
+    config, settings_module = get_config(settings_module)
+    num_pending_plates = 0
+    num_positive_pending_samples = 0
+    mongo_updated = False
+    mlwh_updated = False
+    dart_updated = False
+    try:
+        # Get barcodes of pending plates in DART
+        print("Selecting pending plates from DART...")
+        pending_plate_barcodes = pending_plate_barcodes_from_dart(config)
+        num_pending_plates = len(pending_plate_barcodes)
+        print(f"{len(pending_plate_barcodes)} pending plates found in DART")
+
+        if num_pending_plates > 0:
```

Comment:
Can use the walrus operator here to optimise the printing. I think something like:

```
if (num_pending_plates := len(pending_plate_barcodes) > 0:
    print(f"{len(pending_plate_barcodes)} pending plates found in DART")
    ...
```

Reply:
But I want to print the number of pending plates before evaluating whether any were returned

Reply:
Updated to use as discussed